pyproject.toml

```diff
@@ -158,12 +162,21 @@ ignore = [
     # DoNotAssignLambda
     "E731",
 ]
+exclude = ["*.ipynb"]
+
+[tool.ruff.lint.pydocstyle]
+# Use Google-style docstrings.
+convention = "numpy"
```

Comment:
The convention applied appears to differ from the comment above. Consider aligning one or the other.